model-server/src/main/kotlin/org/modelix/model/server/handlers/ModelReplicationServer.kt

```diff
@@ -275,12 +277,48 @@ class ModelReplicationServer(val repositoriesManager: RepositoriesManager) {
     }
 
     private suspend fun ApplicationCall.respondDelta(versionHash: String, baseVersionHash: String?) {
-        val delta = VersionDelta(
-            versionHash,
-            baseVersionHash,
-            objectsMap = repositoriesManager.computeDelta(versionHash, baseVersionHash),
-        )
-        delta.checkObjectHashes()
-        respond(delta)
+        when (val format = request.queryParameters["format"]) {
+            "versionHash" -> {
+                respondText(versionHash)
+            }
+            "plain" -> {
```

Comment:
What about a test for the new format?

Reply:
The new format is the default one for new versions of the client. The old "JSON" format just exists for not breaking old versions of the client, but should be removed in the future. This means "plain" is already. Testing the deprecated "JSON" format would require adding support for switching the format in the client, which means adding additional deprecated code.

Reply:
Could also be tested with a plain HTTP test towards the server.